### DISCUSSION TITLE
How can we detect paste event in alpine js?

### DISCUSSION THREAD

**Discussion by toriqsetiawan** [alpinejs/alpine#2880] [1. Help] [answered]
Sorry, i just wondering and try to search on the internet how to handle paste event in alpine js, but i have nothing.
i really appreciate for this help.
thanks

**Comment by SimoTod** [comment, marked as answer]
`x-on:paste` (or `@paste` if you use the short version. X-on accepts any standard browser event.

**Reply by toriqsetiawan** [reply]
There was something wrong with my code. thanks mate
